packages/terra-form-textarea/src/terra-dev-site/doc/example/Resizable.jsx

```diff
@@ -9,6 +9,7 @@ const ResizableExample = () => (
       size="small"
       defaultValue="I automatically resize as you type in more information (except in mobile)."
       id="resizable"
+      ariaLabel="text-area-label"
```

Comment:
Even the docs i would rather update only the some examples with this change. let say small, large and medium size examples.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2583/commits/0aa2ae3fc943878309d490d58b2707b61e888845.